introduction.md

```diff
@@ -46,30 +46,10 @@ If someone has done something worthy of a taco all you have to do is mention the
 
 ## Community Driven Projects
 
-Here are a couple of projects we have going on!
-
-These community driven projects are hosted on [GitHub](https://github.com/pyslackers). These projects are headed up by members of our community and are intended to provide value to our slack team and help members grow in their technical abilities. Each project has people who are leads and other members that act at a mentorship level. It's kinda like Xavier's school of gifted children except without a Blackbird SR-71.
+These projects are maintained by members of our community and are intended to provide value to our slack team and help members grow in their technical abilities.Community driven projects are hosted on [GitHub](https://github.com/pyslackers). Each project has people who are leads and other members that act at a mentorship level. It's kinda like Xavier's school of gifted children except without a Blackbird SR-71.
 
 Feel free to join the [#community_projects](https://pythondev.slack.com/messages/C2FMLUBEU) channel and see where you can contribute, all contributions are welcomed from any skill level.  Low hanging fruit with getting started includes testing, documentation, code, feature ideas, bug reports, and front end design. You do not need to be a grandmaster Python jedi in order to contribute to one of the projects. If you are we welcome that too :wink:.
```

Comment:
There is two space before `Low hanging fruit`. And maybe we could rephrase this paragraph saying there is difficulty labels and `help wanted`